recipes/torchsurv/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set name = "torchsurv" %}
+{% set version = "0.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/torchsurv-{{ version }}.tar.gz
+  sha256: 6d91eedb7b45cb4581f88962dfb959a6252d085a998b8985a86286dedbea1621
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+  skip: true  # [win]
```

Comment:
It is OK for noarch package to fails on Windows with a missing dependency. We prefer noarch b/c it is faster to build and the package will be ready when the missing dependency is available on Windows.

Reply:
Reverted to noarch, hope this works!